scipy/_lib/_array_api.py

```diff
@@ -354,3 +354,36 @@ def xp_unsupported_param_msg(param):
 
 def is_complex(x, xp):
     return xp.isdtype(x.dtype, 'complex floating')
+
+
+# temporary substitute for xp.minimum, which is not yet in all backends
+# or covered by array_api_compat.
+def xp_minimum(x1, x2):
```

Comment:
I notice we are accumulating quite a lot of these, I think in terms of the bigger picture (ie not within the scope of this PR) I think it would be quite nice to have a way to document somewhere what we have and their status in the array API (are they intended to be temporary or not)

Reply:
The ones I added here are in the 2023 standard; they just aren't in the libraries or our `array-api-compat` yet.

Reply:
This should materialize in `array-api-compat` within weeks I expect. Fine to commit to our `_array_api.py` now; should be temporary.